static/app/components/replays/player/horizontalMouseTracking.tsx

```diff
@@ -34,22 +48,38 @@ function HorizontalMouseTracking({children}: Props) {
         return;
       }
 
-      const rect = await getBoundingRect(elem.current);
-      const left = e.clientX - rect.left;
-      if (left >= 0) {
-        const percent = left / rect.width;
-        const time = percent * duration;
-        setCurrentHoverTime(time);
-      } else {
-        setCurrentHoverTime(undefined);
+      try {
+        const rect = await getBoundingRect(elem.current, {
+          signal: controller.current?.signal,
```

Comment:
is the `?` needed since we should always have an AbortController in there

```suggestion
          signal: controller.current.signal,
```